packages/view/backend/sirius-components-view-builder/src/main/java/org/eclipse/sirius/components/view/builder/IViewDiagramElementFinder.java

```diff
@@ -0,0 +1,16 @@
+package org.eclipse.sirius.components.view.builder;
```

Comment:
- Missing copyright on tons of files
- Add a CodingRulesTests and ArchitectureConstants as architectural test in order to ensure that the code in this project matches our expectations

Reply:
Should I put copyright in the generated files too ?